Add KUDO Cassandra Platform Service Docs (#3515)
mesosphere/dcos-docs-site: pages/dkp/kommander/1.3/projects/platform-services/federated-addons/index.md

        catalog.kubeaddons.mesosphere.io/origin-repository: https://github.com/mesosphere/kubeaddons-enterprise
        catalog.kubeaddons.mesosphere.io/origin-repository-version: master
        values.chart.helm.kubeaddons.mesosphere.io/jenkins: https://raw.githubusercontent.com/jenkinsci/helm-charts/main/charts/jenkins/values.yaml
      labels:
        kubeaddons.mesosphere.io/name: jenkins
      name: jenkins
    spec:
      chartReference:
        chart: stable/jenkins
        values: |
          ---
          master:
            useSecurity: false
            installPlugins:
              - prometheus:2.0.6
              - kubernetes:1.18.2
              - workflow-job:2.33
              - workflow-aggregator:2.6
              - credentials-binding:1.19
              - git:3.11.0
            csrf:
              defaultCrumbIssuer:
                enabled: false
                proxyCompatability: false
            serviceType: "ClusterIP"
            jenkinsUriPrefix: "/jenkins"
            ingress:
              enabled: true
              path: /jenkins
              annotations:
                kubernetes.io/ingress.class: traefik
        version: 1.9.4
EOF
```

> Ensure the projectns variable is set before executing the command. This variable is the project namespace (i.e. Kubernetes Namespace associated with the project) that was defined/created when the project itself was initially created.

Reviewer: ```suggestion
Ensure the projectns variable is set before executing the command. This variable is the project namespace (that is, the Kubernetes Namespace associated with the project) that was defined/created when the project itself was initially created.
```

> Then, if you run the following command on a Kubernetes cluster associated with the Project, you’ll see a Kubernetes Addon Object, in the corresponding namespace:

Reviewer: ```suggestion
After you run the following command on a Kubernetes cluster associated with the Project, a Kubernetes Addon Object appears in the corresponding namespace:
```
